clang/test/CIR/CodeGen/label-values.c

```diff
@@ -0,0 +1,76 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir  %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s --check-prefix=CIR
+
+void A(void) {
+  void *ptr = &&A;
+A:
```

Comment:
It feels like a bad idea to have the label and function use the same identifier in the test.